packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -0,0 +1,340 @@
+
+class FormAds extends Component {
+    state = {
+        initial_values: {
+            country        : '',
+            currency       : '',
+            type           : '',
+            asset          : '',
+            fix_price      : '',
+            amount         : '',
+            min_transaction: '',
+            max_transaction: '',
+            advertiser_note: '',
+        },
+        is_loading: true,
+    }
+
+    componentDidMount() {
+        // TODO: call api to populate country, currency, and asset
+        WS().send({ 'residence_list': 1 }).then(() => { // this is just to mock the api delay response
+            const new_initial_values = {
+                country        : 'Indonesia',
+                currency       : 'IDR',
+                type           : 'buy',
+                asset          : 'USD',
+                fix_price      : 10000,
+                amount         : 50,
+                min_transaction: 1000,
+            };
+            this.setState({
+                initial_values: new_initial_values,
+            });
+
+            if (this.props.ad_id) {
+                // call the api, get the file based on id
+                // populate the state from the respnose
+            } else {
+                this.setState({ is_loading: false });
+            }
+    
+        });
+    }
+
+    handleSubmit(formik_vars) {
+        // TODO: p2p API call to create ad
+        // eslint-disable-next-line no-console
+        console.log(this.state);
+        // eslint-disable-next-line no-console
+        console.log(formik_vars);
+    }
+
+    render() {
+        return <Fragment>
+            <div className='my-ads__heading--wrapper'>
+                <div onClick={() => this.props.handleShowForm(false)} className='my-ads__heading--btn'>
+                    <IconBack />
+                </div>
+                <h2>{localize('Create new ad')}</h2>
+            </div>
+            {this.state.is_loading ? <Loading is_fullscreen={false} /> : (
+                <Formik
+                    initialValues={{ ...this.state.initial_values }}
+                    onSubmit={this.handleSubmit}
+                    validate={this.validateFormAds}
+                >
+                    {({
+                        isSubmitting,
+                        errors,
+                        touched,
+                        isValid,
+                        setFieldValue,
+                        values,
+                    }) => (
+                        <div className='my-ads__form'>
+                            <Form noValidate>
+                                <ThemedScrollbars
+                                    autoHide
+                                    style={{ height: 'calc(520px - 70px)' }} // height of container minus height of modal footer container
+                                >
+                                    <div className='my-ads__form--container'>
```

Comment:
Why is `container` a modifier (`--`)? Shouldn't it be like `my-ads__form-container`?